polyfill/lib/calendar.mjs

```diff
@@ -273,27 +297,55 @@ impl['gregory'] = ObjectAssign({}, impl['iso8601'], {
     if (!HasSlot(date, ISO_YEAR)) date = ES.ToTemporalDate(date, GetIntrinsic('%Temporal.PlainDate%'));
     return GetSlot(date, ISO_YEAR) < 1 ? 'bc' : 'ad';
   },
-  year(date) {
+  eraYear(date) {
     if (!HasSlot(date, ISO_YEAR)) date = ES.ToTemporalDate(date, GetIntrinsic('%Temporal.PlainDate%'));
     const isoYear = GetSlot(date, ISO_YEAR);
     return isoYear < 1 ? -isoYear + 1 : isoYear;
   },
-
+  year(date) {
```

Comment:
This could just be deleted, and fall back to the one from `impl['iso8601']`, I think?